### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [devlive-community/datacap#449] [Q&A]
# index

DataCap is integrated software for data transformation, integration and visualization.

https://datacap.edurt.io/

**Comment by bindie2005** [comment]
你好，在线示例无法访问，有新的地址吗？

**Reply by qianmoQ** [reply]
目前暂时不提供在线实例了，可以使用 docker 方式进行本地快速构建

**Comment by xiaoyv521** [comment]
这个群失效了可以在换个二维码吗？

**Reply by qianmoQ** [reply]
二维码已经更新，感谢支持

**Comment by sivanxiong** [comment]
新手部署了半个月没成功,谁可以指教一下 1156016884

**Reply by qianmoQ** [reply]
具体是什么方式部署呢？
